backend/src/main/java/com/htecgroup/skynest/service/impl/FileServiceImpl.java

```diff
@@ -239,6 +239,8 @@ public void deleteFile(UUID fileId) {
     if (fileMetadataEntity.isDeleted()) {
       throw new FileAlreadyDeletedException();
     }
+    if (!fileMetadataEntity.getBucket().getIsPublic())
+      permissionService.currentUserHasPermissionForFile(fileMetadataEntity, AccessType.EDIT);
```

Comment:
You forgot to add the curly brackets, I know it's one line and java allows this, but I don't.

Reply:
Yes Master